dora/shaded/client/pom.xml

```diff
@@ -374,9 +367,8 @@
               <arguments>
                 <argument>-fnsv</argument>
                 <argument>build/alluxio-${project.version}-hadoop2-client.jar</argument>
-                <argument>${project.parent.parent.basedir}/client/alluxio-${project.version}-client.jar</argument>
+                <argument>${project.parent.parent.parent.basedir}/client/alluxio-${project.version}-client.jar</argument>
```

Comment:
cross reference with the comment above

Reply:
yea this is just to change the minimal. I don't know maven well enough to refactor this. I guess we can change these two to the same format after my PR is merged? I'd like this change to be in a separate PR instead of a shady commit hidden in this one.

Reply:
sound good to me. not really a blocker for this PR.